parameter-sets/node-pool-request/parameter-set.json

```diff
@@ -86,6 +86,24 @@
             "type": "STRING",
             "mandatory": false,
             "visibilityCondition": "!model.useSameNetworkAsDSSHost"
+        },
+        {
+            "type": "SEPARATOR",
+            "label": "Kubernetes"
+        },
+        {
+            "name": "labels",
+            "label": "Nodepool Labels",
```

Comment:
Since it's a native K8S feature and not an Azure one, better call it "Node labels"